cmd/lncli/types.go

```diff
@@ -3,38 +3,38 @@ package main
 import (
 	"fmt"
 
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/lightningnetwork/lnd/lnrpc"
 )
 
-// OutPoint displays an outpoint string in the form "<txid>:<output-index>".
-type OutPoint string
-
-// NewOutPointFromProto formats the lnrpc.OutPoint into an OutPoint for display.
-func NewOutPointFromProto(op *lnrpc.OutPoint) OutPoint {
-	return OutPoint(fmt.Sprintf("%s:%d", op.TxidStr, op.OutputIndex))
+// newOutPointFromProto formats the lnrpc.OutPoint into a string for display.
+func newOutPointFromProto(op *lnrpc.OutPoint) string {
+	var hash chainhash.Hash
+	copy(hash[:], op.TxidBytes)
+	return fmt.Sprintf("%v:%d", hash, op.OutputIndex)
 }
 
-// Utxo displays information about an unspent output, including its address,
+// utxo displays information about an unspent output, including its address,
 // amount, pkscript, and confirmations.
-type Utxo struct {
+type utxo struct {
```

Comment:
nit: don't see the benefit in making these private, currently it provides a library to other applications that may want to format data in the same way as lncli. also not much use in having a private struct with public fields

Reply:
Reverted the private change.